### PR TITLE
Complete arcane epic

### PR DESCRIPTION
**Pull request by myrlyn**
Fix conflicts between Player's Guide to Faerun and Arms and Equipment Guide.(require AEG for PGF, and comment out items in PGF - Drow Poison, Flight Arrow, Signal Arrow)
Comment out Specific MODs for Energy Resistance Feat, replace with single MOD for CHOOSE based feat in existing LST file. 

Other minor conflict and mod fixes.

### CODE REVIEW

File: data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/races_of_faerun/_races_of_faerun.pcc [hunk 1]
```diff
@@ -7,7 +7,7 @@ GENRE:Fantasy
 BOOKTYPE:Setting Supplement
 SETTING:Fantasy
 STATUS:ALPHA
-PRECAMPAIGN:1,INCLUDES=Player's Guide to Faerun
+PRECAMPAIGN:2,INCLUDES=Player's Guide to Faerun,Arms and Equipment Guide
```
Comment (BahamutDragon) [inline data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/races_of_faerun/_races_of_faerun.pcc]: How does this reconcile with the changes @FerretDave just made?

File: data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/races_of_faerun/rac_equip.lst [hunk 2]
```diff
@@ -39,9 +39,9 @@ Dendritic Crystal																							PROFICIENCY:ARMOR|Dendritic Crystal
...
-Arrow (Flight)						OUTPUTNAME:Arrow, [NAME]																											TYPE:Ammunition.Standard.Wooden.Arrow																									COST:8	WT:0.2																																																SOURCEPAGE:p.155
+#Arrow (Flight)						OUTPUTNAME:Arrow, [NAME]																											TYPE:Ammunition.Standard.Wooden.Arrow																									COST:8	WT:0.2																																																SOURCEPAGE:p.155
 Arrow (Sea)							OUTPUTNAME:Arrow, [NAME]																											TYPE:Ammunition.Standard.Wooden.Arrow																									COST:7	WT:0.2																																																SOURCEPAGE:p.155
-Arrow (Signal)						OUTPUTNAME:Arrow, [NAME]																											TYPE:Ammunition.Standard.Wooden.Arrow																									COST:5	WT:0.2																																																SOURCEPAGE:p.155
+#Arrow (Signal)						OUTPUTNAME:Arrow, [NAME]																											TYPE:Ammunition.Standard.Wooden.Arrow																									COST:5	WT:0.2																																																SOURCEPAGE:p.155
```
Comment (BahamutDragon) [inline data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/races_of_faerun/rac_equip.lst]: See how #418 handled all this.

File: data/35e/wizards_of_the_coast/supplement/deities_and_demigods/dd_templates.lst [hunk 3]
```diff
@@ -2,7 +2,7 @@
...
 # Template Name	Unique Key				Hit Dice Size	Visible	Damage Reduction	Template																										Source Page			Define Stat			Multiple Requirements														Required Race		Combat bonus				Misc bonus		Stat bonus							Modify VAR							Ability																																Spell Res.
-Mummy (Greater)						HITDIE:12		VISIBLE:YES				TEMPLATE:CHOOSE:Slam (Fine)|Slam (Diminutive)|Slam (Tiny)|Slam (Small)|Slam (Medium)|Slam (Large)|Slam (Huge)|Slam (Gargantuan)|Slam (Colossal)	TEMPLATE:Undead	SOURCEPAGE:DD		DEFINESTAT:NONSTAT|CON																						BONUS:COMBAT|AC|8|TYPE=NaturalArmor	BONUS:MISC|CR|2	BONUS:STAT|STR|4	BONUS:STAT|INT,WIS,CHA|2									ABILITY:Internal|AUTOMATIC|Race Traits ~ Greater Mummy|Race Traits ~ Mummy
+Mummy (Greater)						HITDIE:12		VISIBLE:YES				TEMPLATE:CHOOSE:Slam (Fine)|Slam (Diminutive)|Slam (Tiny)|Slam (Small)|Slam (Medium)|Slam (Large)|Slam (Huge)|Slam (Gargantuan)|Slam (Colossal)	TEMPLATE:Undead	SOURCEPAGE:DD		DEFINESTAT:NONSTAT|CON																						BONUS:COMBAT|AC|8|TYPE=NaturalArmor	BONUS:MISC|CR|2	BONUS:STAT|STR|4	BONUS:STAT|INT,WIS,CHA|2									ABILITY:Internal|AUTOMATIC|Race Traits ~ Greater Mummy|Race Traits ~ Greater Mummy
```
Comment (BahamutDragon) [inline data/35e/wizards_of_the_coast/supplement/deities_and_demigods/dd_templates.lst]: Adding the same non-mult ability twice does not really do anything other than waste space.

File: data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/races_of_faerun/_races_of_faerun.pcc (hunk 1 above)
Comment (FerretDave) [inline data/35e/wizards_of_the_coast/campaign_settings/forgotten_realms/races_of_faerun/_races_of_faerun.pcc]: My version means AE is optional, so is prefered.
However i was only looking at dupe equipment, the other fixes may be needed.

Cheers
Dave

**Pull request closed without merging**